pkg/controller/nameservice/nameservice_controller.go

```diff
@@ -216,17 +229,40 @@ func (r *ReconcileNameService) updateNameServiceStatus(instance *rocketmqv1alpha
 		}
 
 		// use admin tool to update broker config
-		if share.IsNameServersStrUpdated && (len(oldNameServerListStr) > cons.MinIpListLength) && (len(share.NameServersStr) > cons.MinIpListLength) {
+		if isNameServersStrUpdated && (len(oldNameServerListStr) > cons.MinIpListLength) && (len(newNameServerListStr) > cons.MinIpListLength) {
+			// bash-4.4$ ./mqadmin clusterList -n 192.168.180.36:9876
+			// #Cluster Name     #Broker Name            #BID  #Addr                  #Version                #InTPS(LOAD)       #OutTPS(LOAD) #PCWait(ms) #Hour #SPACE
+			// broker            broker-0                0     192.168.180.40:10911   V4_5_0                   0.00(0,0ms)         0.00(0,0ms)          0 471030.34 -1.0000
+			// broker            broker-0                1     192.168.137.89:10911   V4_5_0                   0.00(0,0ms)         0.00(0,0ms)          0 471030.34 0.2673
+			clusterListCmd := exec.Command("sh", cons.AdminToolDir, cons.ClusterList, "-n", oldNameServerListStr)
```

Comment:
As `isNameServersStrUpdated` is true, why using `oldNameServerListStr` as the addresses of name servers?